pkgs/applications/science/biology/cd-hit/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, zlib, perl }:
+
+stdenv.mkDerivation rec {
+  version = "4.8.1";
+  name = "cd-hit-${version}";
+
+  src = fetchFromGitHub {
+    owner = "weizhongli";
+    repo = "cdhit";
+    rev = "V${version}";
+    sha256 = "032nva6iiwmw59gjipm1mv0xlcckhxsf45mc2qbnv19lbis0q22i";
+  };
+
+  buildInputs = [ zlib perl ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    PREFIX=$out/bin make install
```

Comment:
Overriding phases breaks features of generic builder. Try using `makeFlags` to set prefix and `preInstall` to create the directory.

Reply:
Fixed!